Packs/CiscoAMP/Integrations/CiscoAMP/CiscoAMP_test.py

```diff
@@ -0,0 +1,1267 @@
+    Scenario:
+    -   Put a computer in isolation.
+    Given:
+    -   The user has entered a connector_guid, comment adn unlock_code.
+    When:
+    -    cisco-amp-computer-isolation-create is called.
+    Then:
+    -   Ensure outputs_prefix is correct.
+    """
+    args: Dict[str, Any] = {
+        'connector_guid': '1',
+        'comment': 'Hello',
+        'unlock_code': 'Goodbye',
+    }
+
+    mock_response = load_mock_response('isolation_response.json')
+    requests_mock.put(
+        f'{BASE_URL}/computers/{args["connector_guid"]}/isolation',
+        json=mock_response
+    )
+
+    from CiscoAMP import computer_isolation_create_command
+    response = computer_isolation_create_command(mock_client, args)
+
+    assert response.outputs_prefix == 'CiscoAMP.ComputerIsolation'
+    assert response.outputs['connector_guid'] == args['connector_guid']
+    assert response.outputs['available'] == mock_response['data']['available']
+    assert response.outputs['status'] == mock_response['data']['status']
+    assert response.outputs['unlock_code'] == mock_response['data']['unlock_code']
+
+
+def test_computer_isolation_delete_command(requests_mock, mock_client):
+    """
+    Scenario:
+    -   Delete a computer in isolation.
+    Given:
+    -   The user has entered a connector_guid.
+    When:
+    -    cisco-amp-computer-isolation-delete is called.
+    Then:
+    -   Ensure outputs_prefix is correct.
+    """
+    args: Dict[str, Any] = {
+        'connector_guid': '1',
+    }
+
+    mock_response = load_mock_response('isolation_response.json')
+    requests_mock.delete(
+        f'{BASE_URL}/computers/{args["connector_guid"]}/isolation',
+        json=mock_response
+    )
+
+    from CiscoAMP import computer_isolation_delete_command
+    response = computer_isolation_delete_command(mock_client, args)
+
+    assert response.outputs_prefix == 'CiscoAMP.ComputerIsolation'
+    assert response.outputs[0]['available'] == mock_response['data']['available']
+    assert response.outputs[0]['status'] == mock_response['data']['status']
+    assert response.outputs[0]['unlock_code'] == mock_response['data']['unlock_code']
+
+
+def test_event_list_command(requests_mock, mock_client):
+    """
+    Scenario:
+    -   Get list of events.
+    Given:
+    -   The user has entered no arguments.
+    When:
+    -    cisco-amp-event-list is called.
+    Then:
+    -   Ensure outputs_prefix is correct.
+    -   Ensure there are no links in the outputs.
+    """
+    mock_response = load_mock_response('event_list_response.json')
+    requests_mock.get(
+        f'{BASE_URL}/events',
+        json=mock_response
+    )
+
+    args: Dict[str, Any] = {}
+
+    from CiscoAMP import event_list_command
+    responses = event_list_command(mock_client, args)
+
+    # Not including last element as it is only a readable output.
+    for response in responses[:-1]:
+        assert response.outputs_prefix == 'CiscoAMP.Event'
+
+        if 'file' in response.outputs:
```

Comment:
why wouldn't it be? is that case tested?

Reply:
added a test for last case (testing the readable_output)